cmd/server/assets/codes/bulk-issue.html

```diff
@@ -0,0 +1,302 @@
+{{define "codes/issue-bulk"}}
+
+{{$hasSMSConfig := .hasSMSConfig}}
+
+<!doctype html>
+<html lang="en">
+
+<head>
+  {{template "head" .}}
+  <script
+    src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/4.0.0/crypto-js.min.js"
+    integrity="sha512-nOQuvD9nKirvxDdvQ9OMqe2dgapbPB7vYAMrzJihw5m+aNcf0dX53m6YxM4LgA9u8e9eg9QX+/+mPu8kCNpV2A=="
+    crossorigin="anonymous"></script>
+</head>
+
+<body id="users-import" class="tab-content">
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <div class="card mb-3 shadow-sm">
+      <div class="card-header">Bulk issue</div>
+      <div class="card-body">
+        {{template "beta-notice" .}}
+
+        {{if not $hasSMSConfig}}
+        <div class="alert alert-danger">
+          <span class="oi oi-warning"></span>
+          No SMS provider is configured for this realm.
+          Please contact a realm administrator to enable this feature.
+        </div>
+        {{end}}
+
+        <p>
+          Use this form to bulk issue codes.
+        </p>
+
+        <pre>Example file contents:
```

Comment:
Pre is whitespace sensitive. This looks kinda weird:

Reply:
<img width="468" alt="x" src="https://user-images.githubusercontent.com/408570/99343068-2c948d00-285b-11eb-9d6e-c10643be0786.png">